docker_images/bertopic/app/pipelines/__init__.py

```diff
@@ -0,0 +1,16 @@
+from app.pipelines.base import Pipeline, PipelineException  # isort:skip
+
+from app.pipelines.audio_classification import AudioClassificationPipeline
+from app.pipelines.audio_to_audio import AudioToAudioPipeline
+from app.pipelines.automatic_speech_recognition import (
+    AutomaticSpeechRecognitionPipeline,
+)
+from app.pipelines.feature_extraction import FeatureExtractionPipeline
+from app.pipelines.image_classification import ImageClassificationPipeline
+from app.pipelines.question_answering import QuestionAnsweringPipeline
+from app.pipelines.sentence_similarity import SentenceSimilarityPipeline
+from app.pipelines.speech_segmentation import SpeechSegmentationPipeline
+from app.pipelines.tabular_classification_pipeline import TabularClassificationPipeline
+from app.pipelines.tabular_regression_pipeline import TabularRegressionPipeline
+from app.pipelines.text_to_speech import TextToSpeechPipeline
+from app.pipelines.token_classification import TokenClassificationPipeline
```

Comment:
You need to remove unused imports I think

Reply:
good catch :)